dll/win32/rpcrt4/rpc_async.c

```diff
@@ -158,3 +158,47 @@ RPC_STATUS WINAPI RpcAsyncCancelCall(PRPC_ASYNC_STATE pAsync, BOOL fAbortCall)
     FIXME("(%p, %s): stub\n", pAsync, fAbortCall ? "TRUE" : "FALSE");
     return RPC_S_INVALID_ASYNC_HANDLE;
 }
+
+/***********************************************************************
+ *           RpcGetAuthorizationContextForClient [RPCRT4.@]
+ * 
+ * Returns the Authz context for an RPC client.
+ * 
+ * PARAMS
+ *  ClientBinding       [I] Binding handle on the server that represents
```

Comment:
Please do not copy wording from MSDN. It's preferable to have no descriptions if you don't want to write your own.

Reply:
Completely rewrote. Thanks.